packages/api/src/ontology/WirePropertyTypes.ts

```diff
@@ -15,6 +15,10 @@
  */
 
 export type WirePropertyTypes =
```

Comment:
Should this be `WireAndStructPropertyTypes`? That way we don't break `WirePropertyTypes`?

Reply:
Talked a bit on slack, just to clarify here as well, the reason you think this is a break is because now instead of just a list of strings, its `SimpleWirePropertyTypes` and the record type? Because technically this boils down to the same thing and is just an additive change, but if someone is using this type, could be confusing since now it referencs something called `SimpleWirePropertyTypes`?